vtex-sites/storecomponents.store
# chore(gatsby-plugin-cms): Improve cms graphql types generation

## What's the purpose of this pull request?
This fixes the breaking changes introduced by https://github.com/vtex/faststore/pull/865. For mor info on why these breaking changes are important: https://github.com/vtex/faststore/pull/865

## How to test it?
This is a code only change, no feature on the website should be changed. For this, make sure all content on the website stays the same from master
